crud/crud.hpp

```diff
@@ -27,9 +27,9 @@ namespace bzn
     class crud final : public bzn::crud_base, public std::enable_shared_from_this<crud>
     {
     public:
-        crud(std::shared_ptr<bzn::storage_base> storage, std::shared_ptr<bzn::subscription_manager_base> subscription_manager);
+        crud(std::shared_ptr<bzn::storage_base> storage, std::shared_ptr<bzn::subscription_manager_base> subscription_manager, std::shared_ptr<bzn::node_base> node);
 
-        void handle_request(const bzn::caller_id_t& caller_id, const database_msg& request, const std::shared_ptr<bzn::session_base>& session) override;
+        void handle_request(const bzn::caller_id_t& caller_id, const database_msg& request, std::shared_ptr<bzn::session_base> session = nullptr) override;
```

Comment:
I thought we were going to remove this default param?

Reply:
Me too. Removed the default parameter in crud.hpp - crud::handle_request

propagated the change through the unit tests:
  crud.handle_request("caller_id", msg); -->. crud.handle_request("caller_id", msg, nullptr);